gitversion.go

```diff
@@ -27,8 +27,13 @@ const (
 	MatchField = 1
 )
 
-// VERSION gets set by the build script via the LDFLAGS
-var VERSION string
+// These variables get set by the build script via the LDFLAGS
+// Detail about these variables are here: https://goreleaser.com/#builds
+var (
+	VERSION = "dev"
```

Comment:
Just wondering why the vars are in all caps here, it's just inconsistent with `launcher` and `log-service`

Reply:
:( I did this because of the `version` folder/module that is brought in.

Reply:
Okay 👍